### PR TITLE
Include vfs plugin in issue template

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ Storage backend (external storage):
 ### Client configuration
 Client version:
 
+Virtual File(No, Suffix, Windows):
```
Comment (michaelstingl) [inline .github/issue_template.md]:
```suggestion
Virtual File enabled (No, Suffix, Windows):
```

**Comment by TheOneRing** [comment]
is it ok now?

**Comment by HanaGemela** [comment]
It mixes too many things into one. I'd say `Virtual files enabled (yes or no)`.
We can guess the plugin from operating system. If that is not enough, I'd add `Virtual files plugin (suffix or Wincfapi)` and added a hint how to find it in General tab

**Comment by TheOneRing** [comment]
I think in general we should just require the info string from the about dialog.
--version should in future report the same string.
This will basically make the whole questionair obsolete.

**Comment by TheOneRing** [comment]
@HanaGemela Instead of that questionnaire I'd prefer to ask users to always share the --version string which will tell us about the used libraries the exact version and the used plugin.
What do you think?

**Comment by HanaGemela** [comment]
@TheOneRing Yeah, it might be easier

**Pull request closed without merging**